### PR TITLE
remove whisper dependency

### PR DESCRIPTION
**Pull request by se3000**
Rewrote Whisper's [BytesToUintBigEndian](https://github.com/ethereum/whisper/blob/e350d60ffa4970ee0458fecab0d16fbcdafe5150/whisper.go#L1043) function but following a style closer to [this SO answer](https://stackoverflow.com/questions/13657232/convert-8byte-to-a-uint64/13657822#13657822) but following the Whisper behavior which allows for overflows. Added a test to document the expected behavior.

Cleaned up go modules and removed dependency acknowledgement.

### CODE REVIEW

File: core/utils/utils.go [hunk 1]
```diff
@@ -745,3 +745,13 @@ func (t *ResettableTimer) Reset(duration time.Duration) {
 	}
 	t.timer = time.NewTimer(duration)
 }
+
+func BytesToUint64(buf []byte) uint64 {
+	var result uint64
+	for _, b := range buf {
+		fmt.Println(uint64(b))
```
Comment (RyanRHall) [inline core/utils/utils.go:752]: :scissors:

File: core/utils/utils.go [hunk 2]
```diff
@@ -745,3 +745,13 @@ func (t *ResettableTimer) Reset(duration time.Duration) {
 	}
 	t.timer = time.NewTimer(duration)
 }
+
+func BytesToUint64(buf []byte) uint64 {
```
Comment (j16r) [inline core/utils/utils.go:749]: Would it be better to use `[4]byte` here?

File: core/utils/utils.go [hunk 3]
```diff
@@ -745,3 +745,13 @@ func (t *ResettableTimer) Reset(duration time.Duration) {
...
+	var result uint64
+	for _, b := range buf {
+		fmt.Println(uint64(b))
+		result = result<<8 + uint64(b)
+		fmt.Println(uint64(b), result)
+	}
+	return result
+}
```
Comment (j16r) [inline core/utils/utils.go:757]: Debugging statement?

**Review by RyanRHall — CHANGES_REQUESTED** [review]

**Comment by j16r** [comment]
Have you seen `binary.LittleEndian.Uint64` ?

File: core/utils/utils.go (hunk 2 above)
Comment (se3000) [reply core/utils/utils.go:749]: We don't have a guaranteed size of `dataLengthBytes`. We could trim/expand it, but this seemed simpler.

**Review by j16r — APPROVED** [review]

**Pull request merged**